src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import com.spikes2212.dashboard.Namespace;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import frc.robot.RobotMap;
+
+import java.util.function.Supplier;
+
+public class Leds extends DashboardedSubsystem {
+
+    private final Namespace colorNamespace = namespace.addChild("led colors");
+
+    public static final int NUMBER_OF_LEDS = 60;
+    private Supplier<Integer> red = colorNamespace.addConstantInt("red value", 0);
+    private Supplier<Integer> green = colorNamespace.addConstantInt("green value", 0);
+    private Supplier<Integer> blue = colorNamespace.addConstantInt("blue value", 0);
+
+    public static Leds getInstance() {
+        if (instance == null) {
+            instance = new Leds("led", new AddressableLED(RobotMap.PWM.LED_PORT),
+                    new AddressableLEDBuffer(NUMBER_OF_LEDS));
+        }
+        return instance;
+    }
+
+    public static Leds instance;
+
+    private final AddressableLED led;
+    private final AddressableLEDBuffer ledBuffer;
+
+    private void updateData() {
+        int rotateValue = 0; // here in case i decide to work on it
+
+        for (int i = rotateValue; i < ledBuffer.getLength() + rotateValue; i++) {
+            if (i % 12 == 3 || i % 12 == 6 || i % 12 == 8 || i % 12 == 11 || i % 12 == 0) {
+                ledBuffer.setRGB(i % 60, 0, 0, 0);
+            } else {
+                ledBuffer.setRGB(i, red.get(), green.get(), blue.get());
+            }
+        }
+        led.setData(ledBuffer);
+    }
+
+    private Leds(String namespaceName, AddressableLED led, AddressableLEDBuffer ledBuffer) {
+        super("led");
```

Comment:
use the namespaceName you receive in the constructor (don't actually change anything, you shouldn't inherit from DashboardedSubsystem anyways)